tcms/rpc/api/testrun.py

```diff
@@ -216,42 +216,14 @@ def update(run_id, values):
     if values.get('product_version') and not values.get('product'):
         raise ValueError('Field "product" is required by product_version')
 
-    form = UpdateForm(values)
+    test_run = TestRun.objects.get(pk=run_id)
+    form = UpdateForm(values, instance=test_run)
+
     if values.get('product_version'):
```

Comment:
check for product, not product_version, that's what is being passed on to populate.